### ISSUE TITLE
Unicode Escapes

### ISSUE THREAD

**Issue by laktak** [alek-sys/sublimetext_indentxml#70]
When I auto indent this JSON:
```
{"string":"text ßüöäÜÖÄ"}
```
I get: 
```
{
    "string": "text \u00df\u00fc\u00f6\u00e4\u00dc\u00d6\u00c4"
}
```
There is no need to escape my unicode characters. Could this be fixed?

**Comment by zhaoxin34** [comment]
I got the problem too !-_-!. Who can please fix the Issue

**Comment by alek-sys** [comment]
Closed with a2994dc19b01d5490fb41eec3189e618b712a9c6
